gammapy/utils/fitting/sampling.py

```diff
@@ -0,0 +1,162 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# helper functions for mcmc smapling
+import numpy as np
+import emcee
+import matplotlib.pyplot as plt
+import corner
+
+__all__ = ["uniform_prior", "run_mcmc", "plot_trace", "plot_corner"]
+
+#TODO: so far only works with a uniform prior on parameters
```

Comment:
Does this still apply? You add a normal_prior here also. It doesn't work? If so, OK to leave the TODO, but please elaborate a bit on why it's not working (or add a test and mark it as `xfail` for now), to help you or others in the future to resolve this TODO.

Reply:
updated the reason of the to do:
`there is no way yet to enter min,mean,max in parameters for normal prior
 lnprob() uses a uniform prior. hard coded for now.`